### PR TITLE
feat: add filters component

### PR DESCRIPTION
**Pull request by Aditi0505**
# Description
- Add filters component

# Screenshot
image
image

### CODE REVIEW

**Review by aniketsingh13 — APPROVED** [review]
look good to me

File: screens/filters/index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="keywords" content="HTML, CSS, JavaScript" />
+    <meta name="author" content="Aditi" />
+    <meta name="description" cotent="Component Library for styling purpose." />
```
Comment (SanketDhabarde) [inline screens/filters/index.html:10]: I think you forgot to change the description

**Review by SanketDhabarde — APPROVED** [review]
Looks good to me

File: screens/filters/index.html (hunk 1 above)
Comment (Aditi0505) [reply screens/filters/index.html:10]: Hey, Thanks for pointing out! I will change it now.

**Pull request merged**